### ISSUE TITLE
feature request: auto pause after each line

### ISSUE THREAD

**Issue by khlsvr** [muzuiget/dualsub-support#576]
This is a great browser addon thank you for making it! I have one adjustment that would make it very powerful for learners.

tldr; A setting with a slider that would pause video after each dialogue/line for 1-20 seconds or indefinitely.

Longer explanation:
There is a website dedicated to learning japanese through subtitles at animelon.com. Basically dualsub can do the same thing, but it has one good feature missing. In animelon it's called "Auto Pause Each Dialogue". What this does is it just pauses the video after each subtitle line. This is good when you are at a level where you need to pause the video a lot. Actually I have an advanced feature in my mind for this one. If there was a setting that allowed you to pause the video after each dialogue/line for: indefinitely OR for x seconds. You could have a slider in settings where you could set it to pause and stay that way, or have a slider to make it pause from 1 seconds to say 20 seconds. This would allow you to have time to view the subtitle lines for lets say 5 seconds and the video would then continue automatically for a short while and pause again for 5 seconds.

**Comment by muzuiget** [comment]
I don't intend to add this feature. This functionality relies on subtitles that are precisely aligned with the text and audio, and on YouTube, many subtitles are automatically generated through speech recognition, making the alignment of audio and subtitle segmentation not very accurate.

When learning a foreign language and if you need to practice listening skills, it's still advisable to use players designed specifically for this purpose, along with audio materials.
